kubectl/client.go

```diff
@@ -168,7 +182,10 @@ func (c *Client) applyKustomize(ctx context.Context, path string, options ApplyO
 	}
 	displayArgs = append(displayArgs, "apply", "-f", "-")
 	displayArgs = append(displayArgs, options.Args()...)
-	kubectlCmd := exec.Command("kubectl", displayArgs...)
+
+	// Add opts that are specific to this client
+	args := append(c.KubeCtlOpts, displayArgs...)
+	kubectlCmd := exec.Command(c.KubeCtlPath, args...)
```

Comment:
```suggestion
	// Add opts that are specific to this client
	displayArgs = append(c.KubeCtlOpts, displayArgs...)
	kubectlCmd := exec.Command(c.KubeCtlPath, displayArgs...)
```
Is there any utility in defining a new `args` variable? I'd just append to the existing `displayArgs`.